Doc/reference/expressions.rst

```diff
@@ -10,14 +10,28 @@ Expressions
 This chapter explains the meaning of the elements of expressions in Python.
 
 **Syntax Notes:** In this and the following chapters, extended BNF notation will
```

Comment:
A link to https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form might be helpful here.